resources/access.rb

```diff
@@ -28,26 +28,25 @@
 
 action :grant do
   with_run_context :root do # ~FC037
-    edit_resource(:template, "#{postgresql_data_dir}/pg_hba.conf") do |new_resource|
+    find_resource(:service, 'postgresql')
+    edit_resource(:template, "#{conf_dir}/pg_hba.conf") do |new_resource|
       source new_resource.source
       cookbook new_resource.cookbook
       owner 'postgres'
       group 'postgres'
-      mode '0600'
```

Comment:
This actually needs to be a string so we don't throw out the leading 0

Reply:
I didn't know. I though it was a good practice to not use string for mode parameter